packages/web/src/components/gcds-card/gcds-card.tsx

```diff
@@ -75,80 +122,104 @@ export class GcdsCard {
     observer.observe(this.el, observerConfig);
   }
 
+  /*
+   * Validate required properties
+   */
+  private validateRequiredProps() {
+    this.validateCardTitle();
+    this.validateHref();
+
+    if (this.errors.includes('href') || this.errors.includes('cardTitle')) {
+      return false;
+    }
+
+    return true;
+  }
+
   async componentWillLoad() {
     // Define lang attribute
     this.lang = assignLanguage(this.el);
 
     this.updateLang();
+
+    this.validateBadge();
+
+    let valid = this.validateRequiredProps();
+
+    if (!valid) {
+      console.error(`${i18n['en'].renderError} | ${i18n['fr'].renderError} `);
```

Comment:
We can take the extra step of removing hard-coded values in the error message (i.e. in this case dropping `\'card-title\' or \'href\''`)
```suggestion
      console.error(`${i18n['en'].renderError} | ${i18n['fr'].renderError} `, this.errors);
```

or display it comma-separated:
```suggestion
      console.error(`${i18n['en'].renderError} | ${i18n['fr'].renderError} `, this.errors.toString());
```

Reply:
That could be confusing since in this case optional properties could be included in the errors. In this use case, `badge` could also be in the errors state

Reply:
Oh that's right. I was hoping there was a way to separate it and reduce hard coded values so we don't need to have to write it out in the error message for all components in the future.

I wonder if we should just drop the property names from the error message like @melaniebmn suggested? Or keep it if we have unit tests that catch the exact error message we'd need to change if ever the required properties are mismatched with the messaging.